openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -355,6 +355,14 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     globalGeometryRules.setString(openstudio::GlobalGeometryRulesFields::RectangularSurfaceCoordinateSystem, "Relative");
     m_idfObjects.push_back(globalGeometryRules);
 
+    {
+      auto zoneAirContaminantBalances = model.getConcreteModelObjects<ZoneAirContaminantBalance>();
```

Comment:
This wasn't the catch all, OS_ZoneAirContaminantBalance was explicitly added to the list of objects to translate:

    result.push_back(IddObjectType::OS_ZoneAirContaminantBalance);

I wonder if the issue was the app, if you enter a schedule name in that text box, it isn't saved to the OSM (since it is not a real schedule) but it looks like it is there until you change tabs